zap/src/main/java/org/zaproxy/zap/extension/anticsrf/ExtensionAntiCSRF.java

```diff
@@ -492,7 +492,26 @@ public String generateForm(int hrefId) throws Exception {
      * @since 2.7.0
      */
     public String generateForm(HttpMessage msg) throws UnsupportedEncodingException {
+        return generateForm(msg, new HashMap<String, String>());
+    }
+
+    /**
+     * Generates a HTML form from the given message.
```

Comment:
and optional parameters.

Reply:
Or actually if we stop passing the map.

`Generates a HTML form from the given message, replacing the action with the specified action URL.`